app/models/job/JobService.scala

```diff
@@ -168,31 +168,32 @@ class JobService @Inject()(wkConf: WkConf,
       } yield ()
     } else Fox.successful(())
 
-  def publicWrites(job: Job)(implicit ctx: DBAccessContext): Fox[JsObject] =
+  def publicWrites(job: Job)(implicit ctx: DBAccessContext): Fox[JsValue] =
     for {
       owner <- userDAO.findOne(job._owner) ?~> "user.notFound"
       organization <- organizationDAO.findOne(owner._organization) ?~> "organization.notFound"
-      resultLink = job.resultLink(organization._id)
-      ownerJson <- userService.compactWrites(owner)
+      ownerEmail <- userService.emailFor(owner)
       creditTransactionBox <- creditTransactionService.findTransactionOfJob(job._id).shiftBox
-    } yield {
-      Json.obj(
-        "id" -> job._id.id,
-        "owner" -> ownerJson,
-        "command" -> job.command,
-        "commandArgs" -> (job.commandArgs - "webknossos_token" - "user_auth_token"),
-        "state" -> job.state,
-        "manualState" -> job.manualState,
-        "latestRunId" -> job.latestRunId,
-        "returnValue" -> job.returnValue,
-        "resultLink" -> resultLink,
-        "voxelyticsWorkflowHash" -> job._voxelyticsWorkflowHash,
-        "created" -> job.created,
-        "started" -> job.started,
-        "ended" -> job.ended,
-        "creditCost" -> creditTransactionBox.toOption.map(t => (t.creditDelta * -1).toString)
+    } yield
+      Json.toJson(
+        JobCompactInfo(
+          id = job._id,
+          command = job.command,
+          organizationId = organization._id,
+          ownerFirstName = owner.firstName,
+          ownerLastName = owner.lastName,
+          ownerEmail = ownerEmail,
+          args = job.args - "webknossos_token" - "user_auth_token",
+          state = job.manualState.getOrElse(job.state),
```

Comment:
`effectiveState` should be usable here, correct?
```suggestion
          state = job.effectiveState,
```